webpack/webpack.config.ts

```diff
@@ -75,9 +75,13 @@ export default (
       assetModuleFilename: '[file][query]',
     },
     devServer: {
-      contentBase: path.join(__dirname, '../dist'),
-      writeToDisk: true,
-      disableHostCheck: true,
+      static: {
+        directory: path.join(__dirname, '../dist'),
+      },
+      devMiddleware: {
+        writeToDisk: true,
+      },
+      allowedHosts: 'all',
```

Comment:
```suggestion
      allowedHosts: 'all',
      client: {
        webSocketURL: {
          hostname: 'localhost',
        },
      },
```

This looks like the fix to get the config page to not throw errors and support hot reloading again.  I found this from [this doc](https://webpack.js.org/configuration/dev-server/) and played with several options until I found something that worked.